trunk/SUAVE/Methods/Weights/Buildups/Electric_Vectored_Thrust/empty.py

```diff
@@ -106,46 +97,49 @@ def empty(config,
     bladeSol    = 0.1                                           # Blade Solidity
     AvgCL       = 6 * Ct / bladeSol                             # Average Blade CL
     AvgCD       = 0.012                                         # Average Blade CD
-   
+
     maxLiftPower    = 1.15*maxLift*(
-                    k*np.sqrt(maxLift/(2*1.225*np.pi*0.8**2)) +
+        k*np.sqrt(maxLift/(2*1.225*np.pi*0.8**2)) +
```

Comment:
What are these numbers?

Reply:
See top of the script in the inputs, they were taken from Project Vahana Conceptual Trade Study. @JTrentSmart  did a good job in documenting where he got these correlations from.

Reply:
Where is this documentation?

Reply:
Documentation Updated

Reply:
Documentation added, I also removed some of the hardcoded numbers